config/main.go

```diff
@@ -37,3 +44,93 @@ var (
 	// if RequirePass is set to an empty string, no authentication is required
 	RequirePass string = constants.EmptyStr
 )
+
+func init() {
+	args := os.Args[1:]
+
+	if len(args) == 1 && args[0] == constants.InitConfig {
+
+		err := os.MkdirAll(filepath.Dir(ConFilePath), 0755)
```

Comment:
Should this be a fatal error? If not shall we have some exit or retry logic here?